NegaMax.java

```diff
@@ -82,15 +98,19 @@ private int negaMax(Game game, int depth, int depthCount, int alpha, int beta, i
 		return max;
 	}
 
-	private int scoreScenarios(Game game, int depth){
-		if(game.isTied()){
-			return 0;
-		}
-		else if(game.weWon()){
-			return 1000/depth;
-		}
-		else{
-			return -1000/depth;
+
+	private static void sortMoves(MoveAndValue [] moves){
```

Comment:
O(n^2)! More efficient (and easier) solution: make MoveAndValue implement comparable (so, write a compareTo method), then do a quick-and-efficient Collections.sort()